Jenkinsfile

```diff
@@ -58,21 +58,21 @@ node {
 
       parallel clangDebugRun: {
         stage("clang-debug:test") {
-          sh "./clang-debug/hyriseTest"
+          sh "./clang-debug/hyriseTest clangDebugRun"
```

Comment:
Shouldn't this be a special folder within the binary's folder? We now create folders `clang-debug`, `clangDebugRun`, `clangDebugRunShuffled`, `clang-debug-sanitizers`, and `clangDebugSanitizers`